### PR TITLE
feat: ci updates

### PR DESCRIPTION
**Pull request by mneri-whispir**
Edit: Fixed the issue on tests being not found:
https://github.com/whispir/whispir-java/actions/runs/4188867670/jobs/7260517201

For the "File renamed without changes", it's because of the location of these files. 
It's been moved from `src/main/test` to `src/test` so the test script would find the tests

~~* Currently the `test` task is being skipped, still trying to find out the difference between this and my testing on my personal repo.~~

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "main": "index.js",
+  "scripts": {
+    "mock:server": "prism mock https://raw.githubusercontent.com/whispir/openapi/fix/prism-spec-update/openapi.yaml"
+  },
+  "dependencies": {
+    "@stoplight/prism-cli": "^4.10.5"
+  },
+  "author": "",
+  "license": "ISC"
+}
```
Comment (richardfontein) [inline package.json:14]: I think we're best not to pollute the Java SDK with NodeJS files - we can instead simply run the following command in the CI workflow:
`npx -y @stoplight/prism-cli@^4.10.5 mock https://raw.githubusercontent.com/whispir/openapi/fix/prism-spec-update/openapi.yaml &`

File: src/main/test/java/com/whispir/client/ApiClientTest.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        try {
+            WorkspaceListParams params = new WorkspaceListParams.Builder().build();
+            WorkspaceCollection workspaces = WorkspacesApi.list(params);
+            WorkspaceRetrieveParams getparams = new WorkspaceRetrieveParams.Builder()
+            .setWorkspaceId(workspaceId)
+            .build();
+            Workspace WORKSPACE = WorkspacesApi.retrieve(getparams);
+            System.out.println(">>>>" + WORKSPACE);
+        } catch (ApiException e) {
+            System.out.println("Exception when calling MessagesApi#getMessageById");
```
Comment (richardfontein) [inline src/main/test/java/com/whispir/client/ApiClientTest.java:40]: The log here is not aligned with the operation - it should instead refer to failed Workspace call

**Review by richardfontein — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -41,8 +41,12 @@ jobs:
           - '18'
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+      - name: Run Mock Server
+        run: npx -y @stoplight/prism-cli@^4.10.5 mock https://raw.githubusercontent.com/whispir/openapi/fix/prism-spec-update/openapi.yaml &
 
+      # - uses: actions/checkout@v2
```
Comment (richardfontein) [inline .github/workflows/ci.yml:49]: Got some commented code left here - can we remove?

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -70,5 +74,6 @@ jobs:
           ./gradlew --version
           echo "JAVA_TEST_HOME=$JAVA_TEST_HOME"
 
+
       - name: Run test suite
-        run: ./gradlew test
+        run: ./gradlew test
```
Comment (richardfontein) [inline .github/workflows/ci.yml:79]: Got some redundant whitespace changes here - let's clean up to keep the PR diff clean

File: .gitignore [hunk 5]
```diff
@@ -22,4 +22,4 @@ build
 bin
 
 # environment files
-.env
+.env
```
Comment (richardfontein) [inline .gitignore:25]: Got a whitespace change, let's also cleanup

**Review by richardfontein — CHANGES_REQUESTED** [review]

**Comment by richardfontein** [comment]
Hey @mneri-whispir , it's not clear why there are so many "File renamed without changes" committed files on the PR. Could you please elaborate?

image

**Comment by mneri-whispir** [comment]
@richardfontein for the  "File renamed without changes", it's because of the location of these files. It's been moved from `src/main/test` to `src/test` so the test script would find the tests

**Review by richardfontein — APPROVED** [review]
LGTM!

**Pull request merged**